### PR TITLE
docs(blog): export to independence blog articles 1-5

### PR DESCRIPTION
**Pull request by pdtyreus**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

First 5 articles in the "Porting a Smart Speaker Voice App to Mobile" series including first 2 tutorials.

**Fixes**: #

### CODE REVIEW

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+Spokestack makes it easy to convert a smart speaker voice app like a Google Action or Alexa Skill to a mobile app. In this tutorial, we’ll go through the entire process of converting a sample Alexa skill into an iOS app using SwiftUI.
+
+We’ve provided some components and boilerplate code so you can follow along more easily. You can download the starting and final versions of the [code here](https://d3dmqd7cy685il.cloudfront.net/docs/minecraft-ios-tutorial.zip).
+
+This tutorial is a direct port of the [Alexa sample skill](https://github.com/alexa/skill-sample-nodejs-howto) from Alexa’s GitHub repository. We stayed as true as possible to the original and didn’t try to improve on it too much at this point.
+
+## Installation and Setup
+
+The easiest way to install Spokestack on iOS is using CocoaPods as described in more detail in the [getting started docs](https://spokestack.io/docs/iOS/getting-started). Download and unzip the [example code](https://d3dmqd7cy685il.cloudfront.net/docs/minecraft-ios-tutorial.zip). In the "start" folder, you’ll see a `Podfile` with the following contents:
```
Comment (timmywil) [inline content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]:
```suggestion
The easiest way to install Spokestack on iOS is using CocoaPods as described in more detail in the [getting started docs](/docs/iOS/getting-started). Download and unzip the [example code](https://d3dmqd7cy685il.cloudfront.net/docs/minecraft-ios-tutorial.zip). In the "start" folder, you’ll see a `Podfile` with the following contents:
```

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+## Make the App Listen
+
+If you run the app now, you’ll see it doesn’t do much. It just displays a screen with a microphone button.
+
+![](screen1.png)
+
+Next, you’re going to start the Spokestack speech pipeline so that your app can hear you. You’re going to configure the app so that it starts listening when you press a button.
+
+The example code contains a class `PipelineStore` that implements [the delegates](https://spokestack.io/docs/iOS/speech-pipeline) in the Spokestack speech pipeline. The `init` method of our `ContentView` already calls the `pipelineStore.start()` method to start background processing. To begin actually listening, add a new function to `PipelineStore`:
```
Comment (timmywil) [inline content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]:
```suggestion
The example code contains a class `PipelineStore` that implements [the delegates](/docs/iOS/speech-pipeline) in the Spokestack speech pipeline. The `init` method of our `ContentView` already calls the `pipelineStore.start()` method to start background processing. To begin actually listening, add a new function to `PipelineStore`:
```

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+    ListeningIcon(isListening: $pipelineStore.isListening)
+        .background(Color.blue)
+        .foregroundColor(Color.white)
+        .cornerRadius(40)
+    }.padding()
+```
+
+Try running the app now. At this point, you should be able to press the button, say something, and see what you said appear in the UI.
+
+![](screen2.gif)
```
Comment (timmywil) [inline content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: Could use an alt.

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+end
+```
+
+Run the command `pod install` to download and install the dependencies. Next, open `Minecraft Skill.xcworkspace` using XCode.
+
+## Make the App Listen
+
+If you run the app now, you’ll see it doesn’t do much. It just displays a screen with a microphone button.
+
+![](screen1.png)
```
Comment (timmywil) [inline content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: Could use an alt for the image.

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+    print(self.currentResponse.speak)
+    DispatchQueue.main.async {
+        self.appSays = self.currentResponse.speak
+    }
+})
+```
+
+At this point your app should be able to understand and answer questions.
+
+![](screen3.gif)
```
Comment (timmywil) [inline content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: alt

**Review by timmywil — CHANGES_REQUESTED** [review]
Some nits

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md (hunk 1 above)
Comment (pdtyreus) [reply content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: fixed

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md (hunk 2 above)
Comment (pdtyreus) [reply content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: fixed

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md (hunk 3 above)
Comment (pdtyreus) [reply content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: added alt

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md (hunk 4 above)
Comment (pdtyreus) [reply content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: added alt

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md (hunk 5 above)
Comment (pdtyreus) [reply content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: added alt

**Review by timmywil — APPROVED** [review]

**Pull request merged**